yksmp/src/lib.rs

```diff
@@ -14,11 +14,15 @@ use std::error;
 struct Function {
     addr: u64,
     record_count: u64,
+    stack_size: u64,
```

Comment:
Can we not make this `usize` and de-x86 it?

Reply:
Just checked, this is already u64 in LLVM, so we inherit the type here. https://llvm.org/docs/StackMaps.html#stackmap-format

Reply:
OK.